engine/swarm/stack-deploy.md

```diff
@@ -8,6 +8,16 @@ When running Docker Engine in swarm mode, you can use `docker stack deploy` to
 deploy a complete application stack to the swarm. The `deploy` command accepts
 a stack description in the form of a [Compose file](../../compose/compose-file/compose-file-v3.md).
 
+> **Note**
+>
+> The `docker stack deploy` command uses the legacy
+> [Compose file version 3](../../compose/compose-file/compose-file-v3.md)
+> format, used by Compose V1. The latest format, used by Compose V2,
```

Comment:
I'd clarify that the latest format is known as the 'Compose spec' or 'Compose specification'